test/GridTest.cpp

```diff
@@ -28,11 +28,22 @@ namespace rtt{
         Grid grid = Grid(2, 2, 1, 1, 4, 4);
         for (auto nestedPoints : grid.getPoints()) {
             for (auto point : nestedPoints) {
-                EXPECT_TRUE(point.x >= 2 && point.x < 3);
-                EXPECT_TRUE(point.y >= 2 && point.y < 3);
+                EXPECT_TRUE(point.x >= 2 && point.x <= 3);
+                EXPECT_TRUE(point.y >= 2 && point.y <= 3);
             }
         }
     }
+    TEST(Grid,getters){
+        Grid grid = Grid(1,2,3,4,5,6);
+        EXPECT_EQ(grid.getOffSetX(),1);
+        EXPECT_EQ(grid.getOffSetY(),2);
+        EXPECT_EQ(grid.getRegionWidth(),3);
+        EXPECT_EQ(grid.getRegionHeight(),4);
```

Comment:
Testing the getters and setters might be a bit overkill, unless they are not pure imo

Reply:
No it's not, I very much disagree. I found your error because I was trying to test them for coverage. This also prevents someone in the future to make the exact same mistake again.